pynopegl-utils/pynopegl_utils/export.py

```diff
@@ -58,27 +59,32 @@ class EncodeProfile:
         name="MP4 / H264 4:2:0",
         format="mp4",
         # Since 4:2:0 is used for portability (over the Internet typically), we also use faststart
-        args=["-pix_fmt", "yuv420p", "-c:v", "libx264", "-crf", "18", "-movflags", "+faststart"],
+        args=["-pix_fmt", "yuv420p", "-crf", "18", "-movflags", "+faststart"],
+        encoder="libx264",
     ),
     mp4_h264_444=EncodeProfile(
         name="MP4 / H264 4:4:4",
         format="mp4",
-        args=["-pix_fmt", "yuv444p", "-c:v", "libx264", "-crf", "18"],
+        args=["-pix_fmt", "yuv444p", "-crf", "18"],
+        encoder="libx264",
```

Comment:
Can you rebase on top of main so it includes the AV1 profile ?

Reply:
Done